### DISCUSSION TITLE
v0.78.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#7278] [Announcements]
### Comment resolving

<img width="737" alt="image" src="https://github.com/user-attachments/assets/31f12e81-61a5-4331-b7a8-6902cbb9dc9e">

It's now possible to resolve comment threads on documents from the comment menu. You can always view and restore a document's resolved comments by clicking the checkmark icon at the top right of the sidebar.

### Icon and emojis

<img width="737" alt="image" src="https://github.com/user-attachments/assets/f1956172-09ec-4b72-85f1-b569234eea38">

Thanks to @hmacr who put in a ton of work to rebuild our picker interface, you can now select emoji and icon for both document and collection – finally enabling emoji on collections and icons on documents – we also added even more new icons to choose from in this release.

### Discord Authentication

Discord was added as a fully supported authentication provider plugin. In addition to the regular security controls for email domain you can also configure to restrict logins to a specific Discord server and roles. See more in the [documentation](https://docs.getoutline.com/s/hosting/doc/discord-g4JdWFFub6).

### Other improvements
- When creating a document inside a collection it is now published by default (#7243)
- A display preference was added to move the table of contents to the right side of documents (#6943)
- Users with "Can edit" permission for a document now have the ability to create child documents. This brings the permission inline with the Edit permission on collections (#7257)
- Frequently used code block languages are now shown first in the list (#7222)
- `AZURE_TENANT_ID` added to allow custom azure tenant (#7028)
- Table of contents is now enabled by default on publicly shared docs with headings
- Images that cannot load now show an improved broken state in the editor (#7200)
- Support for TLDraw viewer links added (#7205)
- Users will no longer be prompted to install the PWA on publicly shared docs (#7198)
- Collection icon can now be directly updated in the same manner as collection (#7120)
- Sharing dialog now has keyboard navigation (#7102)
- API keys now display when they were last used (#7082)
- API keys can now have an expiry set (#7064)
- Full Swedish 🇸🇪  and Norwegian 🇳🇴 translations were added

### Fixes
- Groups added to a collection are no longer always added with `read-only` permission
- Admins were unable to see documents shared from another users private collection (#7264)
- Collection actions available in command menu after navigating away from a collection (#7241)
- "Edit" button and other functionality disappears after unpublishing a document (#7258)
- Table insert controls no longer visible in read only mode (#7249)
- Document icons are now shown in link search results in the editor
- Image and file upload now works correctly on mobile devices (#7161)
- Fixed an issue where the "Archived by" text would temporarily reference the wrong user (#7223)
- Fixed an issue with groups not appearing in collection share options
- Fixed an issue where the table of contents last heading would never be highlighted (#7204)
- Contrast of search highlighting increased in dark mode
- Text will no longer become selected when resizing sidebars
- CJK document content no longer overflows in search results (#7183)
- Whitespace only comments are no longer allowed (#7168)
- Duplicating a document no longer loses advanced formatting
- Iframely is now correctly disabled with no API_KEY in env
- Reduced false positives for "smart" replacement of fractions in editor (#7141)
- Fixed an issue where it was not possible to choose 'No access' as bulk import permission
- Fixed an issue where the comment sidebar could not be scrolled until multiple comments were made (#7112)
- Scrollwheel can no longer cause zoomed images to get stuck (#7083)

## New Contributors
* @sebasptsch made their first contribution in https://github.com/outline/outline/pull/7005
* @mkrfowler made their first contribution in https://github.com/outline/outline/pull/7061
* @MnkyArts made their first contribution in https://github.com/outline/outline/pull/7184
* @airbenich made their first contribution in https://github.com/outline/outline/pull/7028
* @feingold-ant made their first contribution in https://github.com/outline/outline/pull/7243
* @guoard made their first contribution in https://github.com/outline/outline/pull/7246
* @alexislefebvre made their first contribution in https://github.com/outline/outline/pull/7273

**Full Changelog**: https://github.com/outline/outline/compare/v0.77.1...v0.78.0

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.78.0'>v0.78.0</a>.</em>

**Comment by brnbs** [comment]
@tommoor when can we expect the next release? Do you follow any release schedule/plan?

**Reply by Limezy** [reply]
The usual frequency is around one release per month

**Reply by kameleo-team** [reply]
can we expect the next release anytime soon?
